src/main/scala/com/typesafe/sbt/site/paradox/ParadoxSitePlugin.scala

```diff
@@ -10,7 +10,7 @@ import com.typesafe.sbt.web.SbtWeb
 
 /** Paradox generator. */
 object ParadoxSitePlugin extends AutoPlugin {
-  override def requires = SitePlugin && SbtWeb
+  override def requires = SitePlugin && SbtWeb && ParadoxPlugin
```

Comment:
This could be added in the migration guide, that now it is not required to have `ParadoxPlugin` and only `ParadoxSitePlugin` is enough.

Reply:
Adding `ParadoxPlugin` explicitly wasn't required before these upgrades either, but I'm not sure what made that change.

Reply:
Yea, thats right. This was not in the older Paradox version: https://github.com/lightbend/paradox/blob/v0.5.4/plugin/src/main/scala/com/lightbend/paradox/sbt/ParadoxPlugin.scala#L40

Therefore it was not needed to enable `ParadoxPlugin` on a project if one would add settings from `ParadoxPlugin.paradoxSettings` manually, as that was the only thing that the pugin did.

But with the addition of `ParadoxTheme` now the auto-plugin added settings and the config.

I guess we could still mention in the migration guide, that it is not required to explicitly enable `ParadoxPlugin` anymore.